lms/static/js/verify_student/views/webcam_photo_view.js

```diff
@@ -151,20 +154,6 @@
                      return this.imageData;
                  },
 
-                 flashObjectTag: function() {
-                     return (
-                        '<object type="application/x-shockwave-flash" ' +
```

Comment:
This was moved to template and the function is unused now, removing.
https://github.com/edx/edx-platform/blob/b9c9df00c4002ad01c3330e5fd2d566ff904d4a8/lms/templates/verify_student/webcam_photo.underscore#L10-L18

Reply:
Tested flash camera on safari and it's working fine.